app/assets/javascripts/admin/addon/components/admin-plugin-config-page.gjs

```diff
@@ -1,19 +1,19 @@
 import Component from "@glimmer/component";
 import { inject as service } from "@ember/service";
+import HorizontalOverflowNav from "discourse/components/horizontal-overflow-nav";
+import NavItem from "discourse/components/nav-item";
 import i18n from "discourse-common/helpers/i18n";
+import I18n from "discourse-i18n";
 import AdminPluginConfigArea from "./admin-plugin-config-area";
 
 export default class extends Component {
   @service currentUser;
-
-  get configNavRoutes() {
-    return this.args.plugin.configNavRoutes || [];
-  }
+  @service adminPluginConfigNavManager;
```

Comment:
Sounds a bit like word salad. 😅 Not sure if we can cut out any of the words here?

Reply:
Fair enough lol. I'm open to suggestions, but its kinda quite a specific thing. Maybe just `adminPluginNavManager`?

Reply:
Will go with that